src/PrimaryAuthController.js

```diff
@@ -97,6 +97,9 @@ function (Okta, PrimaryAuthForm, CustomButtons, FooterRegistration, PrimaryAuthM
       },
       'focusout input': function (e) {
         $(e.target.parentElement).removeClass('focused-input');
+      },
+      'click .button-show': function () {
```

Comment:
Is there a reason we're using a namespace 'sign-in-widget-event' rather than a named event like 'showPassword'?